### PR TITLE
make a zoo file with different animal classes

### PR DESCRIPTION
**Pull request by GeorgeBatch**

### CODE REVIEW

File: zoo.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+class Animal:
+    def __init__(self, name):
+        self.name = name
+        self.age = 2
+
+    def speak(self):
+        print(self.age)
```
Comment (avaspataru) [inline zoo.py]: You should print("My age is ", self.age)

File: zoo.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    def speak(self):
+        print("I am a dog")
+
+
+class Bear(Animal):
+    def speak(self):
+        print("I am a bear named", self.name)
+
+
+animal = Animal(name='fluffy')
```
Comment (avaspataru) [inline zoo.py]: All names (here and on the lines below) should start with capital letters.

**Review by avaspataru — CHANGES_REQUESTED** [review]

File: zoo.py (hunk 1 above)
Comment (GeorgeBatch) [reply zoo.py]: made the change

File: zoo.py (hunk 2 above)
Comment (GeorgeBatch) [reply zoo.py]: made the change

**Review by avaspataru — APPROVED** [review]
LGTM

**Pull request merged**